InstallGuide.md

```diff
@@ -0,0 +1,49 @@
+Canal provides two things
+
+- Networking between hosts in your cluster
+- Enforcement of network policy
+
+# Networking
+Networking can be handled either by `flannel` or by `calico`. Regardless of which is used, Calico needs to be involved when containers are started. See the [Policy](#policy) section below. 
```

Comment:
Actually, I think removing the "Use Calico for inter-host networking" parts of this doc would help make this quite a bit easier to follow.  You can turn it into well-defined sections:
- Installing Flannel Daemon
  - Choosing a backend
- Installing Calico Daemon
- Installing CNI Plugins + Config
  - IPAM

Eventually, when we've got a closer integration and a "canal" daemon, the "choosing a backend" part can include using BGP.